stories/link-bars/MatTagBarStory.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import { DemoContainer } from '../storybook-components';
+
+import { Heading, HugeHeading } from '../../src/atoms/Heading';
+import { Code } from '../../src/atoms/Code';
+
+import {
+	LinkBarLinkBordered,
+	Row,
+} from '../../src';
+
+import theme from '../../src/themes/mat';
+
+const { colors } = theme;
+
+const MatTagBarStory = () => (
+	<section>
+		<HugeHeading>Tag Filter Bar</HugeHeading>
+		<p>Example of a filter bar that will be used for tags in MAT project. </p>
+		<p>You can specify background color for LinkBarLinkBordered. Default value is theme primary color. </p>
+
+		<Heading>Demo</Heading>
+		<DemoContainer background={colors.white}>
+			<Row
```

Comment:
The other LinkBar* elements are written for HorizontalLinkBar and VerticalLinkBar, yet this one is demoed in a Row instead. Could we switch this to a HorizontalLinkBar?

Reply:
Is there any specific reason to use LinkBar instead of Row? 
For Mat we don't need scrolling behavior, etc

Reply:
I reckon that a bordered link bar button is definitely something someone wants in different project. That's why the component is in shiny in the first place, right?

The reason to use a `*LinkBar` instead of `Row` is for consistency – for sticking to a pattern instead of breaking it. That will make the behavior of the elements more predictable the next time we pick them up.

The specific pattern in this case, is that `LinkBar*` elements go inside `*LinkBar` elements.

Reply:
Now that I'm thinking about it, we'll actually have to stick to that pattern not to break semantics. HorizontalLinkBar is a `ul`, LinkBarLinkBordered is a `li`, and Row is a `div`.

Reply:
Ok, but what should we do about oppskrift usecase?

Reply:
Just the same?

Reply:
Yes - one or more `LinkBarLink`s inside a `HorizontalLinkBar`

Reply:
The only thing I noticed so far is that LinkBar adds scroll instead of making several lines of items

Reply:
Ok, I see two options here:

- either inherit LinkBarBorderedLink from another component, e.g. Button to preserve semantics
- or adjust LinkBarBorderedLink to fit to overall concept of LinkBars, and then add margins at oppskrift side

@misund what do you think?

Reply:
> The only thing I noticed so far is that LinkBar adds scroll instead of making several lines of items

You can override that with `flex-wrap: wrap;`

Reply:
I think the best way to go would be to adjust LinkBarBorderedLink to fit the overall concept of LinkBars, complete with margins and all, and then use them directly on the oppskrift side

Reply:
Seems like an overkill, to be honest. We just don't need LinkBar functionality on oppskrift for tags and I don't see any advantages for adding flex-wrap because this would add unnecessary nesting and styles, yet what we get in the end is just a Row.

Reply:
Upon inspection, I see that HorizontalLinkBar already accepts a `shouldWrap` prop, I'm guessing it's exactly what we need.

Someone has had this need before.